extension-style/src/main/java/com/mapbox/maps/extension/style/layers/generated/ModelLayer.kt

```diff
@@ -237,6 +237,64 @@ class ModelLayer(override val layerId: String, val sourceId: String) : ModelLaye
     setProperty(propertyValue)
   }
 
+  /**
+   * Enable/Disable shadow casting for this layer
+   */
+  val modelCastShadows: Boolean?
```

Comment:
Shouldn't we also mark property MapboxExperimental here as well?

Reply:
The whole `ModelLayer` is marked as experimental, probably redundant to mark individual properties as well then.

Reply:
Makes total sense